index.html

```diff
@@ -0,0 +1,198 @@
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex corporis alias delectus aut aliquid repudiandae
+          quaerat ad hic magnam pariatur quasi accusamus eaque provident, rerum officia sequi, cumque autem iusto!</p>
+      </article>
+      <article>
+        <h2>Monthly Web Development</h2>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex corporis alias delectus aut aliquid repudiandae
+          quaerat ad hic magnam pariatur quasi accusamus eaque provident, rerum officia sequi, cumque autem iusto!</p>
+      </article>
+
+      <article>
+        <h2>Monthly Web Development</h2>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex corporis alias delectus aut aliquid repudiandae
+          quaerat ad hic magnam pariatur quasi accusamus eaque provident, rerum officia sequi, cumque autem iusto!</p>
+      </article>
+
+      <article>
+        <h2>Monthly Web Development</h2>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex corporis alias delectus aut aliquid repudiandae
+          quaerat ad hic magnam pariatur quasi accusamus eaque provident, rerum officia sequi, cumque autem iusto!</p>
+      </article>
+    </div>
+    <div class="seemorearticles">
+      <h3> See more on twitter </h3>
+    </div>
+    <div class="featuresmashingbook">
+      <div class="textwrap">
+        <h1> Design system</h1>
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Et enim animi, nobis quis esse nesciunt excepturi
+          nam
+          velit. Perspiciatis laborum aliquam dolorem eveniet eaque atque dolorum amet sequi debitis exercitationem!</p>
+      </div>
+    </div>
+    <div class="articles">
+        <article>
+          <h2>Monthly Web Development</h2>
+          <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex corporis alias delectus aut aliquid repudiandae
+            quaerat ad hic magnam pariatur quasi accusamus eaque provident, rerum officia sequi, cumque autem iusto!</p>
+        </article>
+  
+        <article>
+          <h2>Monthly Web Development</h2>
+          <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex corporis alias delectus aut aliquid repudiandae
+            quaerat ad hic magnam pariatur quasi accusamus eaque provident, rerum officia sequi, cumque autem iusto!</p>
+        </article>
+  
+        <article>
+          <h2>Monthly Web Development</h2>
+          <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex corporis alias delectus aut aliquid repudiandae
+            quaerat ad hic magnam pariatur quasi accusamus eaque provident, rerum officia sequi, cumque autem iusto!</p>
+        </article>
+      </div>
+    <div class="publicity">
+      <div class="newsletter smashing "><h1>Smashing Newsletter </h1> </div>
+
+      <div></div>
+    </div>
+    <div class="seemorearticles">
+        <h3> See more articles </h3>
+      </div>
+  </main>
+  <footer>
```

Comment:
Your footer should have an `<ul>` tag that contains `<li>` tags positioned with flex or floats, which I see you did but with t he `<a>` tags, please do adjust.